D-docs/02-server/00-index.md

```diff
@@ -229,7 +229,7 @@ This sigchain is from a user who…
 3. Used squares to add another device called “rectangles” with its own key
 4. Used rectangles to follow [cecileb](/cecileb)
 
-You can try browsing a real sigchain [online](/max/sigchain) or through [the API](/_/api/1.0/sig/get.json?uid=dbb165b7879fe7b1174df73bed0b9500). Since sigchains are **public**, you can do this for any user on Keybase!
+You can try browsing a real sigchain [online](https://keybase.io/max/sigchain) or through [the API](/_/api/1.0/sig/get.json?uid=dbb165b7879fe7b1174df73bed0b9500). Since sigchains are **public**, you can do this for any user on Keybase!
```

Comment:
```suggestion
You can try browsing a real sigchain [online](https://keybase.io/max/sigchain) or through [the API](https://keybase.io/_/api/1.0/sig/get.json?uid=dbb165b7879fe7b1174df73bed0b9500). Since sigchains are **public**, you can do this for any user on Keybase!
```